README.md

```diff
@@ -1 +1,3 @@
-# crosswordpuzzle-backend
\ No newline at end of file
+# Crosswordpuzzle-Backend
+This repo serves to persist the crosswordpuzzle data in a db and to communicate with different Microservices.
+It also can check if out of given questions a crosswordpuzzle can be created.
```

Comment:
Please enhance the readme! Especially user- and dev-manual are missing. I was not able to run the code because the database currently needs manual preconfiguration and I den't know which exactly.